app/models/page.rb

```diff
@@ -17,4 +17,29 @@ class Page < ActiveRecord::Base
   validates :slug, uniqueness: true
 
   scope :static, -> { where(type: nil) }
+  scope :primary, -> { static.where('slug <> ? AND slug NOT LIKE ?', '', "%/%") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.